### PR TITLE
feat: add scroll support

### PR DESCRIPTION
**Pull request by dohaki**
Closes ACX-2338 and ACX-2339

Do not merge until tested and scroll support is in production

### CODE REVIEW

File: api/_utils.ts [hunk 1]
```diff
@@ -359,14 +359,14 @@ export const getRouteDetails = (
...
-      address: inputToken.addresses[resolvedOriginChainId],
+      address: utils.getAddress(inputToken.addresses[resolvedOriginChainId]),
     },
     outputToken: {
       ...outputToken,
       symbol: isBridgedUsdc(outputToken.symbol)
         ? _getBridgedUsdcTokenSymbol(outputToken.symbol, destinationChainId)
         : outputToken.symbol,
-      address: outputToken.addresses[destinationChainId],
+      address: utils.getAddress(outputToken.addresses[destinationChainId]),
```
Comment (james-a-morris) [inline api/_utils.ts:369]: OOC was this breaking anything?

File: scripts/generate-routes.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ const enabledMainnetChainConfigs = [
   chainConfigs.MODE,
   chainConfigs.BLAST,
   chainConfigs.LISK,
+  chainConfigs.SCROLL,
```
Comment (james-a-morris) [inline scripts/generate-routes.ts:29]: Are we planning to add Scroll Sepolia?

File: api/_utils.ts (hunk 1 above)
Comment (dohaki) [reply api/_utils.ts:369]: Yea, we had a non-checksummed address in the constants package.

Comment (pxrl) [reply api/_utils.ts:369]: That should be fixed now, if you bump to v3.1.11 :+1:

Comment (dohaki) [reply api/_utils.ts:369]: Nice bumped 👍

File: scripts/generate-routes.ts (hunk 2 above)
Comment (dohaki) [reply scripts/generate-routes.ts:29]: I think we will later if needed

**Review by gsteenkamp89 — APPROVED** [review]

**Pull request merged**